site/docs/guides/structured-output.mdx

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 7
+---
+
+# Structured Output
+
+Structured output generation lets a model produce machine-readable responses that follow a defined format - for example JSON, a regular expression, or an EBNF grammar. OpenVINO GenAI provides a flexible structured output enforcement so the model can generate valid, well‑formed output.
```

Comment:
```suggestion
Structured output generation lets to force a model produce machine-readable responses that follow a defined format - for example JSON, a regular expression, or an EBNF grammar. OpenVINO GenAI provides a flexible structured output enforcement so the model can generate valid, well‑formed output.
```
Models are capable of produce machine-readable responses so they don't need additional letting